src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/innertypelast/InputInnerTypeLastClassCtorsInitBlocks.java

```diff
@@ -0,0 +1,26 @@
+package com.puppycrawl.tools.checkstyle.checks.design.innertypelast;
+
+public class InputInnerTypeLastClassCtorsInitBlocks {
```

Comment:
please rebase, there should be violation that `* Config:` is missed.

Reply:
rebased and config comment added
>please share details on why, we have Check on this now.

Not sure what you mean